### ISSUE TITLE
Unparented Popups do not get added to the Tab Order

### ISSUE THREAD

**Issue by StephenLPeters** [microsoft/microsoft-ui-xaml#2871]
I think this issue is a duplicate of an issue I opened years ago, but I was unable to find it to reference.   This issue manifests in TeachingTip not being added to the TabFocus order. It is however added to the Narrator focus order. TeachingTip currently support using the F6 key to move focus to it.
